src/components/FormHelpMessage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+import colors from '../styles/colors';
+import styles from '../styles/styles';
+import stylePropTypes from '../styles/stylePropTypes';
+
+const propTypes = {
+    /** Error or hint text */
+    message: PropTypes.node,
+
+    /** Children to render next to dot indicator. Ignored when message is not empty */
+    children: PropTypes.node,
+
+    /** Indicates whether to show error or hint */
+    isError: PropTypes.bool,
+
+    /** Container style props */
+    style: stylePropTypes,
+};
+
+const defaultProps = {
+    message: null,
+    children: null,
+    isError: true,
+    style: [],
+};
+
+const FormHelpMessage = (props) => {
+    if (_.isEmpty(props.message) && _.isEmpty(props.children)) {
+        return null;
+    }
+
+    return (
+        <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.mt1, styles.mb1, ...props.style]}>
+            {props.isError && <Icon src={Expensicons.DotIndicator} fill={colors.red} />}
+            {!_.isEmpty(props.message) ? (
+                <Text style={[props.isError ? styles.formError : styles.formHelp, styles.flex1, styles.mb0, styles.ml2]}>
+                    {props.message}
+                </Text>
+            ) : <View style={[styles.flex1, styles.ml2]}>{props.children}</View>}
```

Comment:
Now reverse this condition. Children take priority. When `children` is passed override the `props.message`'s Text block.